main.go

```diff
@@ -124,6 +136,33 @@ func init() {
 		}
 	}
 	numrestored = 0
+	checksum = []byte{0x00}
+}
+
+func daemon() {
+	log.WithFields(log.Fields{"func": "daemon"}).Infof("Starting daemon mode with Config: %+v", brf)
```

Comment:
Don't you need to catch SIGTERM and shut down gracefully if you're running in daemon mode? I think you at least want to avoid shutting down halfway through an upload of your backup to S3.

Reply:
fixed